letsencrypt/models/letsencrypt.py

```diff
@@ -149,7 +158,11 @@ def cron(self):
             crt.write(chain_cert.read())
             chain_cert.close()
             _logger.info('wrote %s', crt.name)
-        self.call_cmdline([
-            'sh', '-c', self.env['ir.config_parameter'].get_param(
-                'letsencrypt.reload_command', ''),
-        ])
+        reload_cmd = self.env['ir.config_parameter'].get_param(
+            'letsencrypt.reload_command', False)
+        if reload_cmd:
+            _logger.info('reloading webserver...')
+            self.call_cmdline(['sh', '-c', reload_cmd])
```

Comment:
Isn't it enough without the `sh -c` part? To allow this module to work outside Linux.

Besides, that `letsencrypt.reload_command` should be group-restricted to admin, to avoid any other user to read/write to it. It's a potential security flaw. It should be warned as such in the README too.